src/content/docs/setup/configuration/commerce-configuration.mdx

```diff
@@ -51,89 +58,322 @@ The `getHeaders` function is a helper function which takes a storefront scope (l
+  - **`x-api-key`** - API key for SaaS services (Catalog Service, Live Search, Product Recommendations). See [Commerce Services Connector](https://experienceleague.adobe.com/en/docs/commerce/user-guides/integration-services/saas) for details.
+  - **`Magento-Environment-Id`** - Connects the storefront to the cloud instance serving it. See [Cloud Environment overview](https://experienceleague.adobe.com/en/docs/commerce-cloud-service/user-guide/project/overview#environment-overview) for details.
+
+</TabItem>
+
+<TabItem label="Adobe Commerce as a Cloud Service (ACCS)">
+
+Adobe Commerce as a Cloud Service uses standard Adobe Commerce header naming conventions but does not require API keys or environment IDs in the default configuration.
+
+```json
+{
+  "headers": {
+    "all": {
+      "Store": "{{STORE_VIEW_CODE}}"
+    },
+    "cs": {
+      "Magento-Customer-Group": "{{CUSTOMER_GROUP_HASH}}",
+      "Magento-Store-Code": "{{STORE_CODE}}",
+      "Magento-Store-View-Code": "{{STORE_VIEW_CODE}}",
+      "Magento-Website-Code": "{{WEBSITE_CODE}}"
+    }
+  }
+}
+```
+
+**Header Scopes:**
+
+- **`all`** - Headers applied to all GraphQL requests
+  - **`Store`** - Store view code for Core GraphQL requests. See [GraphQL Headers](https://developer.adobe.com/commerce/webapi/graphql/usage/headers/) for details.
+
+- **`cs`** - Headers for Catalog Service requests
+  - **`Magento-Customer-Group`** - Customer group hash for pricing and tax calculations. See [Customer groups](https://experienceleague.adobe.com/en/docs/commerce-admin/customers/customer-groups) for details.
+  - **`Magento-Store-Code`** - Store to connect to. See [Create stores](https://experienceleague.adobe.com/en/docs/commerce-operations/configuration-guide/multi-sites/ms-admin#step-3-create-stores) for details.
+  - **`Magento-Store-View-Code`** - Store view for Catalog Service requests. See [Store views](https://experienceleague.adobe.com/en/docs/commerce-admin/stores-sales/site-store/store-views) for details.
+  - **`Magento-Website-Code`** - Website to connect to. See [Create websites](https://experienceleague.adobe.com/en/docs/commerce-operations/configuration-guide/multi-sites/ms-admin#step-2-create-websites) for details.
+
+</TabItem>
+
+<TabItem label="Adobe Commerce Optimizer (ACO)">
+
+Adobe Commerce Optimizer uses different header naming conventions and requires Commerce Optimizer-specific headers as defined in the [Merchandising Services API documentation](https://developer.adobe.com/commerce/services/optimizer/merchandising-services/using-the-api/#headers).
+
+```json
+{
+  "headers": {
+    "all": {
+      "Store": "{{STORE_VIEW_CODE}}"
+    },
+    "cs": {
+      "AC-Environment-Id": "{{TENANT_ID}}",
+      "AC-View-ID": "{{CATALOG_VIEW_ID}}",
+      "AC-Source-Locale": "{{LOCALE}}",
+      "AC-Price-Book-ID": "{{PRICE_BOOK_ID}}",
+      "AC-Policy-{{ATTRIBUTE_CODE}}": "{{ATTRIBUTE_VALUE}}"
```

Comment:
This _could_ be an attribute code, but we should probably align more with what the [docs say](https://developer.adobe.com/commerce/services/optimizer/merchandising-services/using-the-api/#headers): `AC-Policy-{*}`. This is actually the _name_ of the policy's trigger as defined in the UI. It likely will be the attribute name (or some form of it) but it can be whatever the user defines. 

<img width="516" height="337" alt="Image" src="https://github.com/user-attachments/assets/cb4b74d1-c8fb-4992-83d3-dbb5384c84ec" />

Reply:
Do we enforce `AC-Policy` in the name at least? Or is the entire trigger a wildcard?